hi-ml-cpath/src/health_cpath/preprocessing/loading.py

```diff
@@ -192,8 +202,6 @@ def __call__(self, data: Dict) -> Dict:
                 size=(scaled_bbox.h, scaled_bbox.w),
                 level=self.level,
             )
-            mask, _ = self.reader.get_data(mask_obj, **get_data_kwargs)  # type: ignore
```

Comment:
removed mask to free up memory, no neeed to store the mask data, it's like storing a secon wsi

Reply:
Not sure why this change was made. the name of the transform `LoadMaskROId` suggests that mask will be loaded. A mask is much lighter than loading WSI. If there is no mask, this won't work right?

Reply:
Oh in this setting the mask is as large as the whole WSI, remember that these mask are tiffiles and here in this line we load them at the the given level. I don't think it matters much for PANDA for example because the slides are relatively smaller but in the case of very large size (25k, 25k) storing the masks affect the memory consumption quite a lot which prevent us from using all cpu cores for preprocessing

Reply:
Also I read LoadMaskRoid as loading a masked region of interest but not the mask necessarily, do you see any use case where will need the mask to be loaded?

Reply:
Shouldn't we load the mask for compatibility with PANDA experiments?